PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -457,21 +559,21 @@ class HTTPClientTests: XCTestCase {
             expect(requestNumber) == completionCallCount
 
             let json = "{\"message\": \"something is great up in the cloud\"}"
-            return HTTPStubsResponse(data: json.data(using: String.Encoding.utf8)!, statusCode:200, headers:nil)
+            return HTTPStubsResponse(data: json.data(using: String.Encoding.utf8)!, statusCode:200, headers: nil)
                 .responseTime(0.003)
         }
 
         let totalRequests = Int.random(in: 50..<100)
         for requestNumber in 0..<totalRequests {
             self.client.performRequest("POST",
-                                       serially: true,
+                                       performSerially: true,
                                        path: path,
-                                       body: ["requestNumber": requestNumber],
-                                       headers: nil) { (status, data, error) in
+                                       requestBody: ["requestNumber": requestNumber],
+                                       headers: [:]) { (status, data, error) in
                 completionCallCount += 1
             }
         }
-        expect(completionCallCount).toEventually(equal(totalRequests))
+        expect(completionCallCount).toEventually(equal(totalRequests), timeout: .seconds(3))
```

Comment:
I noticed tests were passing in my machine but not in CI. I increased the number of totalRequests to 300 in my machine and they started failing after 1.02 seconds of execution so I assumed the reason why these were failing in CI was timeouts. I increased this to 3 seconds, which should be plenty of time but not too long to make tests take too long.